pnpm-lock.yaml

```diff
@@ -1314,6 +1314,9 @@ importers:
       '@vitest/utils':
         specifier: workspace:*
         version: link:../utils
+      '@wdio/types':
+        specifier: ^8.15.10
+        version: 8.15.10
```

Comment:
> it does look like there are some unrelated packages being updated in the lockfile though

As you are updating only the `@wdio/types` in `package.json`, you should not commit unrelated package updates in `package-lock.json`. Pick only the ones that are related to `@wdio/types` and revert the other lines. 
Once all intentional changes of `package-lock.json` are staged and unrelated ones are reverted, you can use `pnpm i --frozen-lockfile` to see if `package-lock.json` is valid. It will throw error if something is missing.

Reply:
Sorry, I've never had to manually update a lockfile before. I think I got it now though